.github/workflows/codegen.yml

```
@@ -23,6 +23,8 @@ jobs:
           node-version: ${{ matrix.node }}
       - name: Generate stuff
         run: cargo run --package mdsf-codegen
+        # A second run is needed until the readme generation is decoupled from the json-schema
+      - run: cargo run --package mdsf-codegen
```

Maybe something like this

```suggestion
      - name: update readme (temporary hack)
        run: cargo run --package mdsf-codegen
```